spring-cloud-consul-discovery/pom.xml

```diff
@@ -23,6 +23,10 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-consul-config</artifactId>
```

Comment:
`ConsulConfigProperties` resides in the `spring-cloud-consul-config` project, therefore, removing this dependency will make it inaccessible in the `spring-cloud-consul-discovery` project. How do we address this issue?